### PR TITLE
Fixes #8887 feat(nimbus):change adds the newly designed header and footer for the history page

### LINKED ISSUE #8887: Tailwind header template
Create a header template using Tailwind

1. Define the header template and its content
1. Style the template using Tailwind classes
1. Ensure the header template is properly rendered in the base template

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3533)

### PR DESCRIPTION
**Pull request by Avi-88**
**Depends on #9061**

Because

- The new 'history' page also introduces newly designed header and footer styled using tailwind css and fontawesome

This commit

- adds the newly designed header and footer to the new base template

**Screenshot**

header-footer

### CODE REVIEW

File: experimenter/experimenter/templates/experimenter_base.html
Comment (yashikakhurana) [inline experimenter/experimenter/templates/experimenter_base.html:36]: would suggest creating two different header and footer and then import into base

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
Would suggest creating two files header and footer, then import into base

File: experimenter/experimenter/templates/common/footer.html
Comment (brennie) [inline experimenter/experimenter/templates/common/footer.html:1]: Don't need a blank line here.

File: experimenter/experimenter/static/assets/logo.svg
Comment (brennie) [inline experimenter/experimenter/static/assets/logo.svg:74]: SVG is missing a trailing newline.

File: experimenter/experimenter/templates/experimenter_base.html
Comment (brennie) [inline experimenter/experimenter/templates/experimenter_base.html:11]: Shouldnt this be the {% tailwindcss %} template tag from your other PR?

**Review by brennie — CHANGES_REQUESTED** [review]
The templates look great! Nice work!

File: experimenter/experimenter/templates/experimenter_base.html
Comment (Avi-88) [reply experimenter/experimenter/templates/experimenter_base.html:11]: So this is because the other PR was causing integration tests to fail and I was trying to figure out the reason so to get the UI changes reviewed earlier I used a CDN. I'll be replacing this once the other PR is merged, sorry

Comment (brennie) [reply experimenter/experimenter/templates/experimenter_base.html:11]: No need to apologize, I was just confused :)

**Review by brennie — APPROVED** [review]

**Review by yashikakhurana — APPROVED** [review]
all looks good, thank you @Avi-88 👏   lets block this until tailwind PR gets merged they you can remove the CDN

**Comment by eliserichards** [comment]
Blocked on https://github.com/mozilla/experimenter/pull/9061

**Comment by eliserichards** [comment]
This is no longer blocked now that the tailwind stuff is merged :) You can rebase this branch with those changes, fix the conflicts, and then we can merge this! 🎉

**Comment by eliserichards** [comment]
Don't merge yet, waiting on the OK from @Avi-88 👍

**Comment by Avi-88** [comment]
> Don't merge yet, waiting on the OK from @Avi-88 +1

All done ready for merge @eliserichards  :+1:

**Pull request merged**
